render-four.js

```diff
@@ -0,0 +1,47 @@
+export function renderPorsche(cars) {
+
+    const porscheContainerEl = document.createElement('div');
+    const makeEl = document.createElement('h1');
+    const modelEl = document.createElement('p');
+    const colorRedEl = document.createElement('p');
+    const colorBlueEl = document.createElement('p');
+    const colorWhiteEl = document.createElement('p');
+    const colorYellowEl = document.createElement('p');
+    const yearEl = document.createElement('p');
+
+    makeEl.textContent = cars.make;
+    modelEl.textContent = cars.model;
+    colorRedEl.textContent = cars.colors[0];
+    colorBlueEl.textContent = cars.colors[1];
+    colorWhiteEl.textContent = cars.colors[2];
+    colorYellowEl.textContent = cars.colors[3];
+    colorRedEl.style.color = cars.colors[0];
+    colorBlueEl.style.color = cars.colors[1];
+    colorWhiteEl.style.color = cars.colors[2];
+    colorYellowEl.style.color = cars.colors[3];
```

Comment:
This seems very repetitive and would benefit from the use of a loop. You are visiting every item stored inside an array after all... What if the colors array had 1,000 items in it?